x-pack/docs/en/security/authorization/document-level-security.asciidoc

```diff
@@ -3,14 +3,24 @@
 === Document level security
 
 Document level security restricts the documents that users have read access to.
-In particular, it restricts which documents can be accessed from document-based 
-read APIs. 
+In particular, it restricts which documents can be accessed from document-based
+read APIs.
 
 To enable document level security, you use a query to specify the documents that
-each role can access. The document query is associated with a particular data
+each role can access. The document `query` is associated with a particular data
 stream, index, or wildcard (`*`) pattern and operates in conjunction with the
 privileges specified for the data streams and indices.
 
+The specified `query`:
+
+* Expects the same format as if it was defined in the search request
+* Accepts queries written as string values
```

Comment:
It accepts both string and nested JSON, e.g. following synatx also works:
```javascript
POST _security/role/click_role
{
  "indices": [
    {
      "names": [
        "events-*"
      ],
      "privileges": [
        "read"
      ],
      "query": {
        "match": {
          "category": "click"
        }
      }
    }
  ]
}
```

The second example of this page shows an example of using nested JSON format as well.

Reply:
I'll change that bullet to read:
> Accepts queries written as either string values or nested JSON

I'll also add your example to show users the string and JSON examples in tandem.